README.md

```diff
@@ -103,6 +103,36 @@ val adbKeyPair = AdbKeyPair.read(privateKeyFile, publicKeyFile)
 Dadb.create("localhost", 5555, adbKeyPair)
 ```
 
+## Development 
```

Comment:
Could you please put those instruction into a `CONTRIBUTING.md` file instead to make the Readme concise?